recipes/text_extensions_for_pandas/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "text_extensions_for_pandas" %}
+{% set version = "0.1b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CODAIT/text-extensions-for-pandas/archive/{{ version }}.tar.gz
+  sha256: ab14b12dcea4b6928c2e1e5c83e3d23dd2b14baee1954504460ed93dd321405c
+
+build:
+  number: 0
+  skip: true  # [win,py<37]
```

Comment:
You need "and" or "or" the comma won't work there.

Reply:
sure